torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -170,12 +170,6 @@ bool canHandle(Node* node) {
     }
     return true;
   }
-  if (node->kind() == prim::Loop) {
-    return false; // TODO
-  }
-  if (!allShapesAreKnown(node)) {
```

Comment:
I think we are no longer checking if the inputs to a node are completely known, just the outputs: https://github.com/pytorch/pytorch/blob/e88d6ccab82252e98f2ef6b3ac09b9c426938632/torch/csrc/jit/passes/tensorexpr_fuser.cpp#L330

So i'm not sure if this is redundant

Reply:
Hmm, I think you are right. I will keep the check for now (I will probably get rid of it anyway in upcoming changes, but in this PR it's probably still technically needed).